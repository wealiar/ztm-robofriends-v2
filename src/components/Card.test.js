import { shallow } from 'enzyme';
import React from 'react';
import Card from './Card';

it('expect to render Card component', () => {
    expect(shallow(<Card />).length).toBe(1);
});

it('card snapshot test', () => {
    expect(shallow(<Card />)).toMatchSnapshot();
});
