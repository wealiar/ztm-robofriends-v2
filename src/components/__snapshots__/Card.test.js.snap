// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card snapshot test 1`] = `ShallowWrapper {}`;
